src/main/java/nextstep/ladder/service/LadderGameHandler.java

```diff
@@ -0,0 +1,33 @@
+package nextstep.ladder.service;
+
+import nextstep.ladder.domain.Ladder;
+import nextstep.ladder.domain.Participants;
+import nextstep.ladder.view.InputView;
+import nextstep.ladder.view.ResultView;
+
+public class LadderGameHandler {
+    public static void runGame() {
+        ParticipantService participantService = new ParticipantService();
+
+        Participants participants = inputAndRegisterParticipant(participantService);
```

Comment:
다른 로직은 메서드로 추출하지않고 바로 사용하셨는데,  이 메서드만 추출해서 사용하셨네요.
강의에도 나왔던 내용이지만, depth 를 다르게 구현한다면 읽는 사람이 헷갈릴 수 있을것 같아요 🤔

```suggestion
        String inputParticipantName = InputView.inputParticipantName();
        Participants participants = participantService.registerParticipant(inputParticipantName);
```